src/map/skill.cpp

```diff
@@ -1303,6 +1303,16 @@ int skill_additional_effect(struct block_list* src, struct block_list *bl, uint1
 			}
 		}
 
+		// Enchanting Shadow gives a chance to inflict Shadow Wounds to the enemy.
+		if (sc != nullptr) {
+			status_change_entry *sce = sc->data[SC_SHADOW_WEAPON];
+			unit_data *ud = unit_bl2ud(bl);
+
+			if (sce != nullptr && ud != nullptr && rnd_chance(sce->val1, 100)) {
```

Comment:
val1 is used for chance and skill lv now?

Reply:
Shadow Weapon (Enchanting Shadow) is the base skill that needs to be active to send out Shadow Scars to targets. So here sce->val1 is SC_SHADOW_WEAPON's chance which is skill_lv%.